eng/mgmt/ci.buildtools.targets

```diff
@@ -26,7 +26,8 @@
             BuildTasksProjects;
         </BuildProjectDependsOn>
         <BuildToolsPackageDependsOn>
-            $(RestoreProjectDependsOn);
+            $(CleanProjectDependsOn);
```

Comment:
I think this is what is causing the CI to fail and my local builds to also fail. The builds are running in parallel with the others and you are cleaning up some of the output as a result. We should remove this.

Reply:
I undid this change locally and my local builds now pass so I definitely believe this is the issue.